src/test/logical_query_plan/lqp_translator_test.cpp

```diff
@@ -964,4 +969,24 @@ TEST_F(LQPTranslatorTest, CreatePreparedPlan) {
   EXPECT_EQ(prepare->prepared_plan(), prepared_plan);
 }
 
+TEST_F(LQPTranslatorTest, Export) {
+  const auto lqp = ExportNode::make("a_table", "a_file.tbl", FileType::Auto, ValidateNode::make(int_float_node));
+
+  const auto pqp = LQPTranslator{}.translate_node(lqp);
+  const auto exporter = std::dynamic_pointer_cast<Export>(pqp);
+
+  EXPECT_EQ(exporter->type(), OperatorType::Export);
+  EXPECT_EQ(exporter->input_left()->type(), OperatorType::Validate);
+}
+
+TEST_F(LQPTranslatorTest, Import) {
```

Comment:
Not an LQPTranslator thing, but do we have a test that makes sure that after an import, MVCC information is correctly set?

Reply:
I doubt it, so I'm gonna add a case in `ImportTest`.